app/views/ladder/MyMatchesTabView.coco.coffee

```diff
@@ -17,7 +17,7 @@ module.exports = class MyMatchesTabView extends CocoView
     'click .load-more-matches': 'onLoadMoreMatches'
     'click .share-ladder-link-button': 'openShareLadderLinkModal'
 
-  initialize: (options, @level, @sessions) ->
+  initialize: (options, @level, @sessions, @anonymousPlayerName) ->
```

Comment:
i dont think we should be adding @ in function params even though `@level` and `@sessions` already are there 😅 

Or is it some coffee trick for instance variable?

Reply:
seems so, that's just like acoffee trick to simply pass in `this.params` 😂